apps/namadillo/src/App/Layout/ActiveAccount.tsx

```diff
@@ -4,8 +4,8 @@ import { defaultAccountAtom, disconnectAccountAtom } from "atoms/accounts";
 import clsx from "clsx";
 import { useAtomValue } from "jotai";
 import { useLocation, useNavigate } from "react-router-dom";
-import { DisconnectAccountIcon } from "./DisconnectAccountIcon";
-import { SwitchAccountIcon } from "./SwitchAccountIcon";
+import { DisconnectAccountIcon } from "../Icons/DisconnectAccountIcon";
```

Comment:
Once they're not in the same folder, can we use `App/Icons/DisconnectAccountIcon` instead?

Reply:
oh, good observation! actually, I'll move it to the same folder as I saw this pattern on other features as well :)